app/src/main/java/io/swagslash/settlersofcatan/Player.java

```diff
@@ -95,6 +96,25 @@ public void setInventory(Inventory inventory) {
         this.inventory = inventory;
     }
 
+    /*
+    addVictoryPoints is used to increase the victory Points of this Player
+     */
+    public void addVictorPoint() {
+        this.victoryPoints++;
```

Comment:
Des is a bisl suboptimal, wir haben bewusst darauf verzichtet mitzuzählen, da man Sachen wie "Längste Handelstraße" wieder verlieren kann etc. Aber is okay atm.